src/source.h

```diff
@@ -132,6 +132,22 @@ class Source : public cyclus::Facility,
   }
   double longitude;
 
+  #pragma cyclus var { \
+    "default": 0, \
+    "uilabel": "Boolean to have source inventory buffer.", \
+    "doc": "The user can set this variable to true to have the source " \
+           "save its maximum throughput for later demand." \
```

Comment:
This documentation string contains a lot of words that don't add information. Perhaps.

"If true, the facility will stockpile its maximum throughput at each time step. If false, the facility will not store inventory."